### ISSUE TITLE
[mesheryctl] Improve mesheryctl Integration testing 

### ISSUE THREAD

**Issue by piyushsingariya** [meshery/meshery#4755]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Desired Behavior

Currently we only just run the `system` commands as the part of Integration Tests, The tests should be improved in-order to cover several E2E things within these integration tests.

#### Implementation

Few vectors we can think 
- Check if meshery is running after running `start` subcommand under `system`.

#### Mockups

---

#### Contributor Resources

- ⌨️ [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by haanhvu** [comment]
@leecalcote @alphaX86 hey I think I can work on this now.

In general, we need to improve the `meshery/mesheryctl/internal/cli/root/system/system_test.go`, right?

> Few vectors we can think
> - Check if meshery is running after running start subcommand under system.

I can start working on this. Along the way we can think of other necessary things to test.

**Comment by alphaX86** [comment]
@haanhvu certainly! Go ahead! You can start working on the idea you have right now

**Comment by leecalcote** [comment]
Excellent. 🙂
